Segfault on launch in noteoftheday::NoteOfTheDay::get_note_by_date

--- origsrc/gnote-0.7.1/src/addins/Makefile.am	2009-12-28 05:46:01.000000000 -0600
+++ src/gnote-0.7.1/src/addins/Makefile.am	2010-01-31 13:17:45.617090200 -0600
@@ -6,7 +6,6 @@ SUBDIRS = backlinks \
 	exporttohtml \
 	fixedwidth \
 	inserttimestamp \
-	noteoftheday \
 	printnotes \
 	stickynoteimport \
 	tomboyimport \
